test/debian-buster/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM debian:buster
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+      procps          \
+      curl            \
+      gnupg           \
+      build-essential
+
+ADD skroutz-stable.list /etc/apt/sources.list.d/
+ADD skroutz-pu.list /etc/apt/sources.list.d/
+RUN curl -sSL http://debian.skroutz.gr/skroutz.asc | apt-key add -
+
+RUN apt-get update &&                       \
+    apt-get install -y librdkafka-dev       \
+                       golang               \
+                       go-dep               \
+                       ruby-full            \
+                       bundler              \
+                       confluent-kafka-2.11 \
+                       openjdk-11-jre
```

Comment:
We can use a single Dockerfile with an `ARG` `$dist`. In order to overcome the `openjdk` version we can use the `default-jre` package instead which depends on the default `openjdk-X-jre` package in each distribution.